scripts/management/create_mgmt_db.py

```diff
@@ -35,17 +35,16 @@ def create_mgmt_db(realisations, f, srf_files=[]):
 
     if len(realisations) == 0:
         print("No realisations found - no entries inserted into db")
-
```

Comment:
Please reinstate whitespace separation between code blocks. The double white space is good between functions.